### PR TITLE
Don't expect symbol loading errors in E2E tests

### PR DESCRIPTION
**Pull request by karupayun**
In this PR we are modifying E2E tests to not expect error boxes when
loading symbols. This is a change that will appear in the next release
as part of the automatic symbol loading feature.

Instead we will check a weaker condition: "At least one module has been
loaded".

Bug: http://b/236928772

Test: Symbol cache E2E is passing locally (in main).

### CODE REVIEW

**Review by reichlfl — APPROVED** [review]
The change itself LGTM, but we should remove all the unneeded code from this test in the near future. The whole part of parsing and collecting the error messages is no longer relevant and just adds noise to the code. Can you do that in a follow-up PR or create a bug for it?

**Review by antonrohr — APPROVED** [review]
LGTM. 

Regarding deletion of code. There is the test case `orbit_symbol_locations.py`, which attempts to load symbols, expects failing, changes a symbol location and loads symbols again, which succeeds. This is a case where checking and dismissing the error is probably needed.

File: contrib/automation_tests/test_cases/symbols_tab.py [hunk 1]
```diff
@@ -120,6 +120,10 @@ def _load_all_modules(self):
         self._modules_dataview.get_item_at(0, 0).click_input('right')
         self.find_context_menu_item('Load Symbols').click_input()
 
+    def _verify_at_least_one_module_is_loaded(self, modules: List[Module]):
+        loaded_modules = [module for module in modules if module.is_loaded]
```
[the remark this answers is not in the document]
Comment (karupayun) [reply contrib/automation_tests/test_cases/symbols_tab.py:124]: It's copy pasted from the function below (line 130) :)

**Comment by karupayun** [comment]
Perfect, what I expected. I had once one error in capture_repeatedly E2E tests and for now I will leave everything as it is. I created this bug: http://b/236965643

**Pull request merged**